students/wachulec_maria/lesson_03_functions/the_collatz_sequence_and_input_validation.py

```diff
@@ -0,0 +1,15 @@
+def collatz(number):
+    if number % 2 == 0:
+        print(number // 2)
+        return number // 2
+    elif number % 2 == 1:
+        print(3 * number + 1)
```

Comment:
it would be better to move `print` after line 13
So there will single print after `collatz` function